.gitguardian.yaml

```diff
@@ -0,0 +1,3 @@
+matches-ignore:
```

Comment:
suggestion: try renaming to `gitguardian.yml` as per [the documentation](https://docs.gitguardian.com/internal-repositories-monitoring/gg_shield/configuration#configuration-file).

Reply:
Spot on! I'm pretty sure I copied the file name from the documentation somewhere. They have a typo as well :) 

But that doesn't seem to have an effect as well. In fact, the GitGuardian link in the checks section points to the previous commit.